frameworks/elastic/universe-kibana/config.json

```diff
@@ -14,6 +14,12 @@
           "description": "The user that runs the Kibana services and owns the Mesos sandbox.",
           "type": "string",
           "default": "nobody"
+        },
```

Comment:
That is a good question. Depends how the Marathon app is configured I think? I think Marathon supports users. Not sure tho.